### ISSUE TITLE
Windows 10

### ISSUE THREAD

**Issue by juliocarraro** [ashkulz/NppFTP#105]
After installing NppFTP on a Windows 10, we have had huge slowness problem when entering a folder via FTP. It also fails every time to download. We have a collaborator distance using Windows 10 in another city, another network, and had no such problems. What may be happening, if you previously were using Windows 7 in the same network and did not have problems? Can be firewall? Some configuration? Any idea what could be the problem?

**Comment by chcg** [comment]
- Which version of nppFTP and n++ is used? 
- Is this issue also happening with another FTP client like e.g. filezilla?
- Is it a plain ftp connection or ftps/ftpes/sftp?

**Comment by juliocarraro** [comment]
Sorry for not replying that I already resolved, but I even forgot that I did this topic.

After a whole day testing the various problems (Firewall, Windows services, reinstalling Notepad++ several times, testing other FTPs), I decided to test the FTP types and solved the problem with explicit FTP.

I did not understand why but solved it. I did not understand why I have a colleague who uses Windows 10, Notepad ++ and NppFTP and he has not configured it until today, uses normal FTP.

Thanks for the answer.
